ui/lib/core/addon/components/replication-page.js

```diff
@@ -40,6 +39,19 @@ export default Component.extend({
     const { model } = this;
     return model.replicationAttrs.mode;
   }),
+  isLoadingData: computed('clusterMode', 'model.{replicationAttrs}', function() {
+    // if clusterMode is bootstrapping
```

Comment:
nit: these comments are very helpful -- could you move them down to line 50 so they're next to the conditional that they're referring to?